.circleci/config.yml

```diff
@@ -75,7 +75,7 @@ jobs:
           name: Running PHPUnit
           working_directory: *PLUGIN_PATH
           command: |
-            composer install && npm install && npm run build
+            composer remove --dev phpunit/phpunit && composer install && npm install && npm run build
```

Comment:
@kienstra on these `npm install` steps... thoughts on switching them to `npm ci` instead? That would ensure the packages installed and used during the build process match the ones in the lock file, as opposed to some of them possibly being updated and different from what you and others have used and tested locally.

Reply:
Ah. good idea. I'll think about that and respond here.

Reply:
That's a good idea, I'll change it to `npm ci`.

Reply:
It's changed in https://github.com/studiopress/genesis-custom-blocks/pull/18/commits/ab549624a5bb64c804a8256fb7808f8490dbe605